Equal heights: Vary example 1's row heights

EricDunsworth opened the pull request:
The demo page's first example demonstrates the plugin's basic concept, but re-uses the exact same content in all three rows. In large view, that gives off the impression that the plugin is unable to distinguish "virtual" rows.

In practice, the plugin is sophisticated-enough to automatically detect new rows and keep track of varying row heights. So why not show it off?

This changes the first example as follows to better demonstrate that capability:
* Place all rows in the same ``wb-eqht`` container
* Make the first row's paragraphs 4 lines apiece
* Halve the amount of content lines in each subsequent row in large view
* Use unique headings for each row
* Note: Shortened French content to get the same number of content lines as the English version in large view

The first two examples in the "pure CSS" equal heights demo have also been updated to match:
* Same changes as the normal equal heights demo
* Reduce demo's custom CSS ``padding`` from ``15px`` to ``10px`` (to prevent the French examples' H3 headings from spanning across multiple lines in large view)

**Screenshots...**
* **English:**
  * **Equal height - example 1**
    | Before | After |
    | --- | --- |
    | ![equalheight-example1-en-before](https://github.com/wet-boew/wet-boew/assets/1907279/afdb1364-3bb1-4c7a-9020-cd3e4f767cb7) | ![equalheight-example1-en-after](https://github.com/wet-boew/wet-boew/assets/1907279/1d27ca44-6baf-4a88-ad32-0a322d4c34dc) |
  * **Equal height (pure CSS) - example 1**
    | Before | After |
    | --- | --- |
    | ![eqht-css-example1-en-before](https://github.com/wet-boew/wet-boew/assets/1907279/d6e61c96-ea52-4396-9cfb-57a09ec9a2c4) | ![eqht-css-example1-en-after](https://github.com/wet-boew/wet-boew/assets/1907279/ca881c31-427b-464e-ae0d-81ea985ae832) |
  * **Equal height (pure CSS) - example 2**
    | Before | After |
    | --- | --- |
    | ![eqht-css-example2-en-before](https://github.com/wet-boew/wet-boew/assets/1907279/8288ed3e-5a97-424d-a0b3-530b0ba50db1) | ![eqht-css-example2-en-after](https://github.com/wet-boew/wet-boew/assets/1907279/e987fffc-9644-4278-a37c-a3fd269e33c2) |
* **Français:**
  * **Égalisation des hauteurs - exemple 1**
    | Avant | Après |
    | --- | --- |
    | ![equalheight-example1-fr-before](https://github.com/wet-boew/wet-boew/assets/1907279/ee95f9fe-fee6-451a-9831-064b27af4d8b) | ![equalheight-example1-fr-after](https://github.com/wet-boew/wet-boew/assets/1907279/f3abbab9-de5d-4b10-a23c-f90ea49331a0) |
  * **Égalisation des hauteurs (pure CSS) - exemple 1**
    | Avant | Après |
    | --- | --- |
    | ![eqht-css-example1-fr-before](https://github.com/wet-boew/wet-boew/assets/1907279/4844e649-e17d-4c7f-a6f2-9f7f7c78e1ba) | ![eqht-css-example1-fr-after](https://github.com/wet-boew/wet-boew/assets/1907279/f8c7f1b6-745d-4449-90b3-8b0d1fef40ec) |
  * **Égalisation des hauteurs (pure CSS) - exemple 2**
    | Avant | Après |
    | --- | --- |
    | ![eqht-css-example2-fr-before](https://github.com/wet-boew/wet-boew/assets/1907279/e6b89dc9-2acd-412b-97b3-1d802281edc8) | ![eqht-css-example2-fr-after](https://github.com/wet-boew/wet-boew/assets/1907279/0c3ecfd9-df1e-47a6-b30f-a1ed4e0be4b4) |

duboisp reviewed (commented):
pre-approved after a local testing.

This is only a content change

EricDunsworth commented:
@duboisp Mind if I port this to [equal heights CSS' demo page](https://wet-boew.github.io/v4.0-ci/demos/eqht-css/eqht-css-en.html) too? Its first example only has like 1.5 rows in large view, but this kind of change would be able to more clearly demonstrate varying row heights.

duboisp commented:
@EricDunsworth you can port it. Just send me a message when it is ready for review. This is a content change so we can merge at anytime once review and tested.

duboisp reviewed (dismissed):
Tested the following and work as expected.

* equalheight/equalheight-en.html
* equalheight/equalheight-fr.html

duboisp commented:
We are just waiting for an additional example with the equal height pure CSS plugin.

* [ ] add a similar working example for the equal height pure CSS plugin

EricDunsworth committed:
Equal heights: Vary example 1's row heights

The demo page's first example demonstrates the plugin's basic concept, but re-uses the exact same content in all three rows. In large view, that gives off the impression that the plugin is unable to distinguish "virtual" rows.

In practice, the plugin is sophisticated-enough to automatically detect new rows and keep track of varying row heights. So why not show it off?

This changes the first example as follows to better demonstrate that capability:
* Place all rows in the same wb-eqht container
* Make the first row's paragraphs 4 lines apiece
* Halve the amount of content lines in each subsequent row in large view
* Use unique headings for each row
* Note: Shortened French content to get the same number of content lines as the English version in large view

The first two examples in the "pure CSS" equal heights demo have also been updated to match:
* Same changes as the normal equal heights demo
* Reduce demo's custom CSS padding from 15px to 10px (to prevent the French examples' H3 headings from spanning across multiple lines in large view)

duboisp commented:
Pre-approved upon successful local testing + second code review

duboisp reviewed (approved):
Reviewed and tested. work as expected and I confirm this is only a content change.
